dist/node/client.js

```diff
@@ -271,19 +271,44 @@ var StitchClient = function () {
         args[_key - 1] = arguments[_key];
       }
 
+      return this._doFunctionCall({
```

Comment:
Can you remove the dist from this PR so we can get a minimal view of what work was done? We will update the dist later.